setup.py

```diff
@@ -36,7 +36,7 @@ def build_ext(*args, **kwargs):
 from setuptools import setup, Extension
 from octobot.constants import PROJECT_NAME, VERSION
 
-PACKAGES = find_packages(exclude=["tests"])
```

Comment:
Why not ignoring tests?

Reply:
If we ignore tests, the ```python setup.py test``` command can't work, either we remove this command or we let the tests folder.